README.md

```diff
@@ -316,6 +316,23 @@ The Kafka message will have a header of source_ip=12.7.0.0.1.
 
 The configuration format is jsonpath. It is descibed in https://docs.fluentd.org/plugin-helper-overview/api-plugin-helper-record_accessor
 
+#### Excluding fields
+Fields can be excluded from output data. Only works for kafka2 and rdkafka2 output plugin.
+
+Fields must be specified using an array of dot notation `$.`, for example:
```

Comment:
`$[...][...]` notation also can be handled in `record_accessor`.
But this notation is confusable. I'm OK to leave as-is.